types/staking.go

```diff
@@ -8,11 +8,11 @@ const (
 
 	// default maximal voting power ratio
 	// Max percentage of the total voting power a single validator can have
-	DefaultMaxVotingPowerRatio = "0.1"
+	DefaultMaxVotingPowerRatio = "0.2"
 
 	// Maximum amount of staking power we allow per validator before the max voting power ratio is enforced
 	// Threshold for number of token staked = DefaultPowerReduction * DefaultMaxVotingPowerEnforcementThreshold
-	DefaultMaxVotingPowerEnforcementThreshold uint64 = 10000
+	DefaultMaxVotingPowerEnforcementThreshold uint64 = 1000000
```

Comment:
Allows for more flexibility for our chains, 10k is the default we stake in most of our scripts for external validators